temp_library/helpers.py

```diff
@@ -3,117 +3,103 @@
 
 
 def openscrn():  # for my personal game-change to your liking
-    """
-    opening screen
-    """
-    print("          /\\\n         /**\\\n        /****\   /\\\n       /      \ /**\\\n      /  /\    /    \ \n     /  /  \  /      \\\n    /  /    \/ /\     \\\n   /  /  _   \/  \/\   \\\n__/__/__( )__/___/__\___\_")
-    print('welcome to the text adventure: "the warlock of firetop mountain"')
-    input("press enter to start your quest...")
+    opening_screen = ""
+
+    opening_screen += "          /\\\n         /**\\\n        /****\   /\\\n       /      \ /**\\\n      /  /\    /    \ \n     /  /  \  /      \\\n    /  /    \/ /\     \\\n   /  /  _   \/  \/\   \\\n__/__/__( )__/___/__\___\_"
+    opening_screen += '\nwelcome to the text adventure: "the warlock of firetop mountain"'
+
+    return opening_screen
+
 
 
 def statbuild():  # inserts initial values to attributes-change the strings and values according to your specific traits
     """stats setting function
 
     :rtype: Trait
     """
-
     trait1 = temp_library.trait.Trait("Skill", 6, 1)
     trait2 = temp_library.trait.Trait("Luck", 6, 1)
     trait3 = temp_library.trait.Trait("Stamina", 12, 2)
 
     # trait1
-    print("to set your "+trait1.getName()+", please role "+str(trait1.getDiceNum())+" die and add "+str(trait1.getConst()))
+    print("to set your "+trait1.get_name()+", please role "+str(trait1.get_dice_num())+" die and add "+str(trait1.get_const()))

```

Comment:
We will have to work on this code. Right now leave it as is. But start thinking about how you would change this, and what exactly can be improved, ok?